lib/clAdapters/google/errors.ts

```diff
@@ -0,0 +1,40 @@
+export abstract class GoogleError extends Error {
+  constructor(message: string);
+  constructor(error: Error);
```

Comment:
nit: since we only have one parameter, the first two constructor declarations are made redundant by the third (since we don't have different return types based on parameters).